scraper/src/libretexts2zim/entrypoint.py

```diff
@@ -46,11 +49,18 @@ def main() -> None:
 
     parser.add_argument(
         "--output",
-        help="Output folder for ZIMs. Default: /output",
-        default="/output",
+        help="Output folder for ZIMs. Default: output",
+        default=os.getenv("LIBRETEXTS_OUTPUT", "output"),
         dest="output_folder",
     )
 
+    parser.add_argument(
+        "--tmp",
```

Comment:
we should probably use tempfile if not specified

Reply:
Fixed